test/lit/passes/precompute-gc.wast

```diff
@@ -0,0 +1,35 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --remove-unused-names --precompute-propagate -all -S -o - \
+;; RUN:   | filecheck %s
+
+(module
+ ;; CHECK:      (func $test-fallthrough (result i32)
+ ;; CHECK-NEXT:  (local $x funcref)
+ ;; CHECK-NEXT:  (local.set $x
+ ;; CHECK-NEXT:   (block (result funcref)
+ ;; CHECK-NEXT:    (drop
+ ;; CHECK-NEXT:     (call $test-fallthrough)
+ ;; CHECK-NEXT:    )
+ ;; CHECK-NEXT:    (ref.null func)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (i32.const 1)
+ ;; CHECK-NEXT: )
+ (func $test-fallthrough (result i32)
+  (local $x funcref)
+  (local.set $x
+   ;; the fallthrough value should be used (which is possible as with names
```

Comment:
This comment looks cut off.

Reply:
Oh, heh, that's what was explaining `--remove-unused-names`, fixing...